rigex-front-end/src/components/passwordInput/PasswordInput.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {
+  Input,
+  IconButton,
+  FormControl,
+  InputLabel,
+  Grid,
+} from '@material-ui/core';
+import {
+  Lock,
+  VisibilityOutlined,
+  VisibilityOffOutlined,
+} from '@material-ui/icons';
+
+const PasswordInput = ({
+  handleChange,
+  handleShowPasswordClick,
```

Comment:
The password being visible seems like something that could be handled on its own inside this component with its own state, instead of relying on a parent component. What do you think?